experimental/packages/opentelemetry-sdk-metrics/src/export/PeriodicExportingMetricReader.ts

```diff
@@ -86,17 +87,19 @@ export class PeriodicExportingMetricReader extends MetricReader {
     }
 
     return new Promise((resolve, reject) => {
-      this._exporter.export(resourceMetrics, result => {
-        if (result.code !== ExportResultCode.SUCCESS) {
-          reject(
-            result.error ??
-            new Error(
-              `PeriodicExportingMetricReader: metrics export failed (error ${result.error})`
-            )
-          );
-        } else {
-          resolve();
-        }
+      api.context.with(suppressTracing(api.context.active()), () => {
```

Comment:
It might be worth extracting this common exporter pattern to @opentelemetry/core to share with `SimpleSpanProcessor` and `BatchSpanProcessorBase`.

```ts
// core
export interface Exporter<T> {
  export(arg: T, callback);
}
export function export<T>(exporter: Exporter<T>, arg: T): Promise<void> {
  return new Promise((resolve, reject) => {
    api.context.with(suppressTracing(api.context.active()), () => {
      exporter.export(arg, result => {...});
    });
  }
}

// reader
return core.export(this._exporter, resourceMetrics);
```

Reply:
Added shared pattern